test/e2e/performanceprofile/testdata/render-expected-output/manual_tuned.yaml

```diff
@@ -22,8 +22,9 @@ spec:
       Different values will override the original values in parent profiles.\n\n[variables]\n#>
       isolated_cores take a list of ranges; e.g. isolated_cores=2,4-7\n\nisolated_cores=1\n\n\nnot_isolated_cores_expanded=${f:cpulist_invert:${isolated_cores_expanded}}\n\n[cpu]\n#>
       latency-performance\n#> (override)\nforce_latency=cstate.id:1|3\ngovernor=performance\nenergy_perf_bias=performance\nmin_perf_pct=100\n\n\n[service]\nservice.stalld=start,enable\n\n\n[vm]\n#>
-      network-latency\ntransparent_hugepages=never\n\n\n[irqbalance]\n#> Override
-      the value set by cpu-partitioning with an empty one\nbanned_cpus=\"\"\n\n\n[scheduler]\nruntime=0\ngroup.ksoftirqd=0:f:11:*:ksoftirqd.*\ngroup.rcuc=0:f:11:*:rcuc.*\nsched_min_granularity_ns=10000000\nsched_migration_cost_ns=5000000\nnuma_balancing=0\n\nsched_rt_runtime_us=-1\n\n\ndefault_irq_smp_affinity
+      network-latency\ntransparent_hugepages=never\n\n\n[irqbalance]\n#> Disable the
+      plugin entirely. It can be racy if NTO restarts for whatever reason. We depend
```

Comment:
nit: `s/if NTO restarts/if TuneD/ restarts/` should be more precise.  NTO (operator) restarting will have no effect on this.

Reply:
right, fixed. Thanks!